src/targets/gitRepository.ts

```diff
@@ -0,0 +1,150 @@
+import simpleGit from 'simple-git';
```

Comment:
please add tests for this new target

Reply:
one other thing -- I think this might be too generically named (it's for a very specific extract-code-to-repository rather than something generically for managing a git repo)